app/views/widgets/onlineTestProgress.scala.html

```diff
@@ -253,7 +253,7 @@
                                             Continue section</button>
                                     }
                                 }
-                                <p>(You'll have 30 minutes to finish)</p>
+                                <p>(The answers that you give are timed)</p>
```

Comment:
This seems direct duplication of text from line 239. I guess @charge-valtech needs to give the content a sprinkle of his magic